tests/pairedomics/test_antismash_downloader.py

```diff
@@ -0,0 +1,245 @@
+from pathlib import Path
+from nplinker.pairedomics.podp_antismash_downloader import GenomeStatus
+from nplinker.pairedomics.podp_antismash_downloader import \
+    _get_genome_status_log
+from nplinker.pairedomics.podp_antismash_downloader import \
+    podp_download_and_extract_antismash_data
+from nplinker.utils import list_files
+
+
+def test_genome_status(tmp_path):
+    genome_status_file = Path(tmp_path, "genome_status.csv")
+    raw_genome_id1 = "GCF_000515175.1"
+    raw_genome_id2 = "GCF_000514635.1"
+    genome_obj1 = GenomeStatus(raw_genome_id1, "None")
+    genome_obj2 = GenomeStatus(raw_genome_id2, "None")
+    assert genome_obj1.resolved_refseq_id == ""
+    assert genome_obj2.resolved_refseq_id == ""
+    genome_status = _get_genome_status_log(genome_status_file)
+    assert isinstance(genome_status, dict)
+    assert len(genome_status) == 0
+    genome_obj1.to_csv(genome_status_file)
+    genome_obj2.to_csv(genome_status_file)
+    genome_status = _get_genome_status_log(genome_status_file)
+    assert isinstance(genome_status[raw_genome_id1], GenomeStatus)
+    assert isinstance(genome_status[raw_genome_id2], GenomeStatus)
+    assert genome_status[raw_genome_id1].original_id == raw_genome_id1
+    assert genome_status[raw_genome_id2].original_id == raw_genome_id2
+
+
+def test_default(tmp_path):
+    download_root = tmp_path / "download"
```

Comment:
The purpose of the `test_default` is not clear to me, I guess you treat multiple records as default. If it's true, better to rename it `test_multiple_records`.